website_snippet_country_dropdown/tests/__init__.py

```diff
@@ -0,0 +1,5 @@
+# -*- coding: utf-8 -*-
+# © 2016 Jairo Llopis <jairo.llopis@tecnativa.com>
+# License LGPL-3.0 or later (http://www.gnu.org/licenses/lgpl).
+
+from . import test_snippet_country_dropdown
```

Comment:
Remember to add a newline character at the end of all files. Usually you can configure your text editor to do it automatically.